docs/testcases/UI/HCS/2766.md

```diff
@@ -0,0 +1,28 @@
+# [Manual] Video from images stack
+
+Test verifies that User can view the same field but in separate time points to compare some parts of the image as a short video and download it.
+
+**Prerequisites**:
+- User that has full permissions to the storage with HCS images
+- HCS image (`image1`) with only one timepoint in the image
+- HCS image (`image2`) with several timepoints in the image
```

Comment:
Now, the ability to create a video for HCS image with several Z-planes is implemented.
Suggest to add following checks:
- creation of video for HCS image with several Z-planes
- switch of video regimens for HCS images that have several timepoints and several Z-planes simultaneously